README.md

```diff
@@ -27,47 +33,44 @@ carpenterd(app)
 
 ```
 
-## Description
-
-`carpenterd-worker` is responsible for receiving job messages from [`carpenterd`][carpenterd], fetching the fully built `npm` tarball from a place like amazon s3, execute the`webpack`/`browserify`/`babel` build using [`workers-factory`][workers-factory].
-
 ## Dependencies
 
 - [`nsq`][nsq] for job distribution to all the workers in given cluster.
-- amazon s3 or an s3 like store for storing built assets and for fetching tarballs stored by [`carpenterd`][carpenterd].
-- A [`cassandra`][cassandra] cluster that is storing the `warehouse.ai` system data.
-
+- amazon s3 or an s3 like store for storing built assets and for fetching
+  tarballs stored by [`carpenterd`][carpenterd].
+- A [`cassandra`][cassandra] cluster that is storing the `warehouse.ai`
```

Comment:
Do we want this to say a `NoSQL database` like carpenterd did? or should that change happen later?